### ISSUE TITLE
content-box vanished after minify

### ISSUE THREAD

**Issue by philipp90** [jakubpawlowicz/clean-css#435]
css:
```
div {
  background: #000 content-box;
}
```
output:
```
div{background:#000}
```

"content-box" is vanished after minifying if written in short notation.

The following example works properly:

css:
```
div {
  background: #000;
  background-clip: content-box;
}
```
output:
```
div{background:#000;background-clip:content-box}
```

**Comment by jakubpawlowicz** [comment]
Please use `--skip-shorthand-compacting` switch until it's fixed.

**Comment by jakubpawlowicz** [comment]
Fixed in master, to be released in 3.1.
